src/Adapter/Tools.php

```diff
@@ -121,11 +121,11 @@ public function bcadd($left_operand, $right_operand, $scale = null)
 
     public function purifyHTML($html, $uri_unescape = null, $allow_style = false)
     {
-        return Tools::purifyHTML($html, $uri_unescape, $allow_style);
+        return LegacyTools::purifyHTML($html, $uri_unescape, $allow_style);
     }
 
     public function refreshCaCertFile()
     {
-        Tools::refreshCaCertFile();
+        LegacyTools::refreshCaCertFile();
```

Comment:
Sometimes you set `<class name>Legacy`, and sometimes `Legacy<class name>`.

This would be better to have the same format everywhere.